backend/corpora/parliament/utils/field_defaults.py

```diff
@@ -69,23 +69,23 @@ def country():
         es_mapping=keyword_mapping(),
     )
 
-def date():
+
+def date(min_date: datetime = MIN_DATE, max_date: datetime = MAX_DATE):
     "The date on which the debate took place."
     return FieldDefinition(
-        name='date',
-        display_name='Date',
-        description='The date on which the debate took place.',
+        name="date",
```

Comment:
Autoformat I suppose? The rest of the file is in single quotes. Don't really need to revert but pointing out for future reference.